platform/broadcom/sonic-platform-modules-cel/services/platform_api/sonic_platform/common.py

```diff
@@ -166,7 +167,7 @@ def write_txt_file(self, file_path, value):
         return True
 
     def is_host(self):
-        return os.system(self.HOST_CHK_CMD) == 0
+        return subprocess.call(self.HOST_CHK_CMD) == 0
```

Comment:
Please consider changing the is_host function implementation as suggested above.